### ISSUE TITLE
Powershell wrapper for CLI

### ISSUE THREAD

**Issue by gregorybrzeski** [hyperonecom/h1-cli#73]
Powershell also known as `PS` is a default MS Windows shell, which is used by MS Windows administrators for scripting and management.

CLI can be used on MS Windows however:

- CLI can return `json` as an output but it is not able to return PS object, needs to be converted using `ConvertFrom-JSON`
- CLI has own command syntax which is not in line with PS   

Some preliminary ideas for conversion of CLI command syntax to PS:

| CLI command | PS Command |
| :------ | :----- |
| h1 vm create .... | new-h1vm ... |
| h1 vm show --vm ... | get-h1vm --vm .... |
| h1 vm list | get-h1vm |
| h1 vm delete --vm .... | remove-h1vm --vm ... |

Some basic information how to build Powershell Modules:
https://kevinmarquette.github.io/2017-05-27-Powershell-module-building-basics/

**Comment by PrzemyslawKlys** [comment]
I can try to help with this. I would need some test account / access so I can see how things work wiith current command line. 

My idea in my current knowledge of product would be to use binary h1 (I assume this works by installing node.js as shown here - https://blog.teamtreehouse.com/install-node-js-npm-windows and you would run it `node.exe h1.js` - sorry I've not played with **NPM** before. If it's run differently on Windows just give me some basic use case. 

I would try to make it compliant with Window starting with PowerShell 5.1. I can try to add 6.0+ support as long as it's behavior is the same on Linux/Mac (returning data in same way). Otherwise it would be additional workload.

**Comment by ad-m** [comment]
@PrzemyslawKlys , the use of CLI does not require download and installation of NodeJS. We bundle our version of NodeJS during release. For development NodeJS environment is required.

See installation instruction at https://github.com/hyperonecom/h1-cli#instalacja . It require only download binary & run.

For account please create account at https://panel.hyperone.com/signup

**Comment by PrzemyslawKlys** [comment]
@ad-m  Right, where do I get CLI binary for Windows then ? If I want to run this on Windows (which is not in the installation instructions)? Or do you mean it is **required** to install NodeJS (your message says it's not required - so a bit confusing).

**Comment by ad-m** [comment]
@PrzemyslawKlys , the binary is available at https://github.com/hyperonecom/h1-cli/releases/tag/v1.1.0 . Using CLI does not require installl NodeJS, just download our binary. Our binary includes our NodeJS in itself.
